include/wlr/types/wlr_xcursor_manager.h

```diff
@@ -50,4 +51,10 @@ struct wlr_xcursor *wlr_xcursor_manager_get_xcursor(
 void wlr_xcursor_manager_set_cursor_image(struct wlr_xcursor_manager *manager,
 	const char *name, struct wlr_cursor *cursor);
 
+/**
+ * Get the name of the cursor image for the given edges.
+ */
+const char *wlr_xcursor_manager_get_resize_name(enum wlr_edges edges);
```

Comment:
Should this be in `wlr_xcursor` instead?

Reply:
I put it here to be as close as possible to the thing that uses it. Nothing in wlr-xcursor uses this. I'll give you the final decision on it.

Reply:
Well, nothing in the xcursor manager uses this. The xcursor manager is only an abstraction around `wlr_xcursor_theme` and `wlr_cursor`.

You could use `get_resize_name` to get the name of the cursor and then call `wlr_xcursor_theme_get_cursor`. If a compositor doesn't want to use `wlr_xcursor_manager`, it could want to use `get_resize_name` with `wlr_xcursor_theme`.

Reply:
Ok that makes sense.